apps/frontend/src/app/3_organisms/ProposalVotingPower/ProposalVotingPower.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FC, useMemo } from 'react';
+
+import { t } from 'i18next';
+
+import { HelperButton, Paragraph, ParagraphSize } from '@sovryn/ui';
+
+import {
+  ProposalProps,
+  ProposalState,
+} from '../../5_pages/BitocracyPage/BitocracyPage.types';
+import { useProposalStatus } from '../../5_pages/BitocracyPage/hooks/useProposalStatus';
+import { VP } from '../../5_pages/StakePage/StakePage.constants';
+import { useGetPersonalStakingStatistics } from '../../5_pages/StakePage/components/PersonalStakingStatistics/hooks/useGetPersonalStakingStatistics';
+import { useGetStakingStatistics } from '../../5_pages/StakePage/components/StakingStatistics/hooks/useGetStakingStatistics';
+import { useAccount } from '../../../hooks/useAccount';
+import { translations } from '../../../locales/i18n';
+import { decimalic, fromWei } from '../../../utils/math';
+import { LabeledAmount } from './ProposalVotingPower.utils';
+
+export const ProposalVotingPower: FC<ProposalProps> = ({ proposal }) => {
+  const { account } = useAccount();
+  const { votingPower } = useGetPersonalStakingStatistics();
+  const { totalVotingPower } = useGetStakingStatistics();
+
+  const status = useProposalStatus(proposal);
+
+  const isVotingPowerVisible = useMemo(
+    () => status === ProposalState.Active,
+    [status],
+  );
+
+  const votingPowerShare = useMemo(() => {
+    if (!votingPower || !totalVotingPower) {
+      return 0;
+    }
+    const votingPowerDecimal = decimalic(votingPower.toString());
+    const totalVotingPowerDecimal = decimalic(totalVotingPower.toString());
+    const getVotingPowerShare = votingPowerDecimal
+      .div(totalVotingPowerDecimal)
+      .mul(100);
+    return getVotingPowerShare.toNumber();
+  }, [votingPower, totalVotingPower]);
+
+  return !isVotingPowerVisible ? (
```

Comment:
This should be visible only if voting is active?
```suggestion
  return isVotingPowerVisible ? (
```

Additionly, i think you should move check to the parent component, so this component would be never loaded if proposal is not active anymore (to prevent unnessesarry calls using hooks to retrieve voting power)